style.css

```diff
@@ -48,28 +56,31 @@
   height: 3px;
   background-color: #6070ff;
   margin: 5px;
+  display: flex;
 }
 
 .first-section {
-  width: 375px;
-  height: 640px;
+  min-height: 640px;
   display: flex;
   flex-direction: column;
-  padding: 100px 24px;
-  border-bottom-left-radius: 15%;
-  background-image: url(images/header-shapes\ mobile.png);
-  background-size: contain;
+  padding: 136px 24px;
+  background-image: url(images/mobile/header-shapes\ mobile.png);
+  background-size: cover;
+  background-position: initial;
   border-bottom-left-radius: 100px;
+  background-color: #fff;
 }
```

Comment:
- [ ] Remember one of the goal of the project is to parse a figma design as close as possible, so please make sure this top section is all aligned properly please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230734612-754cb84a-93c6-44b7-b455-8965d2783366.png)

figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230734642-cbdfde63-8768-4911-b275-1cfba50aa189.png)

Reply:
- [ ] **[OPTIONAL]** Great job working on the suggested changes!  🚀 It would be great if you can add some spacing between the intro paragraph and the `Let's connect` title, to match the design: 🤓 🔧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/230744130-be381ee0-546a-44f6-bbd5-a71a946b342b.png) | ![image](https://user-images.githubusercontent.com/77069011/230744124-8a832009-170d-4d85-8ab7-0d6b0960a90f.png) |